pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix

```diff
@@ -4,7 +4,7 @@
 
 let
 
-  version = "5.4.0.1";
+  version = "8.8.76.61628";
```

Comment:
To my knowledge this is the testing version of skype.

Reply:
Hmm, arch is using the [same](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=skypeforlinux-bin). But you are right, it is still beta. Does this mean you want me to rename the package as "skypeforlinux-unstable"?

Reply:
The problem for me was that the old version could not be installed anymore.

Reply:
I just started skype and can login without problems. We could add a seperate version for unstable though.

Reply:
Which version? 5.4.0.1? Did you reinstalled it?

Reply:
5.4.0.1 on nixpkgs master. I also removed all settings related to skype, but it still works.

Reply:
Ohh, I see what I have done wrong... I tried with nix-env and that does not use my nixpkg checkout :/ That tried to install 5.3.0.1 and that did not worked.

Reply:
`5.4.0.1` should be also in 17.09.